maintainer/CI/build_cmake.sh

```diff
@@ -84,7 +84,7 @@ if [[ ! -z ${with_coverage+x} ]]; then
   bash <(curl -s https://codecov.io/env) &> /dev/null;
 fi
 
-cmake_params="-DCMAKE_BUILD_TYPE=$build_type -DPYTHON_EXECUTABLE=$(which python$python_version) -DWARNINGS_ARE_ERRORS=ON -DTEST_NP:INT=$check_procs $cmake_params"
+cmake_params="-DCMAKE_BUILD_TYPE=$build_type -DPYTHON_EXECUTABLE=$(which python$python_version) -DWARNINGS_ARE_ERRORS=ON -DTEST_NP:INT=$check_procs $cmake_params -DWITH_SCAFACOS=ON"
```

Comment:
I don't think you can do `-DWITH_SCAFACOS=ON` in the build script for all builds as there are containers where there is now scafacos; on these the build should fail now.